e2e/test/scenarios/onboarding/navbar/whats-new.cy.spec.js

```diff
@@ -0,0 +1,84 @@
+import { navigationSidebar, restore } from "e2e/support/helpers";
+import {
+  createMockVersionInfo,
+  createMockVersionInfoRecord as mockVersion,
+} from "metabase-types/api/mocks";
+
+describe("nav > what's new notification", () => {
+  beforeEach(() => {
+    restore();
+
+    mockVersions({
+      currentVersion: "v0.48.0",
+      versions: [
+        mockVersion({
+          version: "v0.48.0",
+          announcement_url: "metabase.com/releases/48",
+        }),
+        mockVersion({
+          version: "v0.47.1",
+          announcement_url: "metabase.com/releases/47",
+        }),
+      ],
+    });
+  });
+
+  // replace `skip` with `only` to manually test this feature with the mocks set up
+  it.skip("mock setup for manual tests", () => {
```

Comment:
Please remove this before merging

Reply:
Should I remove it before merging in the feature branch or before merging in master?